### PR TITLE
chore(deps): update eslint config and fix issues

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: addon-mirage-support/models/file.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+import { Model } from "ember-cli-mirage";
+
+export default Model.extend({});
```
Comment (luytena) [inline addon-mirage-support/models/file.js:3]: Does ember-cli-mirage support native class syntax?

File: addon/mirage-graphql/serializers/file.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import BaseSerializer from "./base";
+
+export default class extends BaseSerializer {
```
Comment (luytena) [inline addon/mirage-graphql/serializers/file.js:3]: Maybe add a class name

**Review by luytena — COMMENTED** [review]
Otherwise looks good to me :ok_hand:

File: addon-mirage-support/models/file.js (hunk 1 above)
Comment (anehx) [reply addon-mirage-support/models/file.js:3]: Sadly no. `.extend` is a custom implementation by mirage

File: addon/mirage-graphql/serializers/file.js (hunk 2 above)
Comment (anehx) [reply addon/mirage-graphql/serializers/file.js]: Done

**Review by luytena — APPROVED** [review]

**Review by czosel — APPROVED** [review]
Nice!

**Pull request merged**